pkg/apisix/cluster.go

```diff
@@ -157,6 +160,7 @@ func (c *cluster) syncCache(ctx context.Context) {
 				zap.String("cost_time", time.Since(now).String()),
 				zap.String("cluster", c.name),
 			)
+			c.metricsCollector.IncrSyncSuccess("cache")
```

Comment:
We don't need this as the counter was increased in `syncOnce`.

Reply:
I kept this counter in `syncOnce` and remove the one in regularly function.